### DISCUSSION TITLE
Calling toHtml on Media model doesn't return alt, unless defined via attributes

### DISCUSSION THREAD

**Discussion by kikky7** [spatie/laravel-medialibrary#3538] [Q&A]
If I don't specify `alt` within attributes, then output has no `alt` tag, looking into `responsiveImageWithPlaceholder.blade.php`, there is also no `alt` attribute. Isn't `name` column here to be used as `alt`?

`$media = $this->getFirstMedia('image');`

```
return $media
            ->img()
            ->attributes([
                'class' => 'w-full h-full object-cover object-center',
                'alt' => $media->name, // without this no alt in the output
            ])
            ->toHtml();
```

**Comment by A909M** [comment]
I've submitted a PR to resolve this: [#3765 ](https://github.com/spatie/laravel-medialibrary/pull/3765).
